### PR TITLE
doc: add digitalocean instruction and better permissions instructions

### PR DESCRIPTION
**Pull request by tangjeff0**

### CODE REVIEW

**Comment by tangjeff0** [comment]
[update self-hosted.readme](https://trello.com/c/yeMS2cDA/38-update-self-hostedreadme)

File: README.self-hosted.md [hunk 1]
```diff
@@ -71,26 +71,31 @@ start the server and Bob's your unkle.
...
-If any of the services fails to launch, you can use `docker compose logs SERVICE_NAME` to inspect what the problem is.
+If any of the services fails to launch, you can use `docker-compose logs SERVICE_NAME` to inspect what the problem is. You can also run `docker-compose ps`  to see all running services. You should see that services `fluree`, `athens`, and `nginx` are up and `healthy`.
 
 The `fluree` service can fail to launch if it does not have enough permissions for the `./athens-data` folder.
-You can work around this particular failure more by manually creating the data folder via `mkdir -p ./athens-date/fluree`.
+You can work around this particular failure more by manually creating the data folder via 
+
+```
+mkdir -p ./athens-data/fluree
+chmod -R 777 athens-data/fluree
```
Comment (filipesilva) [inline README.self-hosted.md:92]: We should add a note here about how this command gives all users of the machine readwrite permissions to the folder. Maybe we can instead give ownership of this folder to the user that the fluree docker image is using? I think it's user id 1000.

File: README.self-hosted.md [hunk 2]
```diff
@@ -100,10 +105,34 @@ or via an `.env` file located in the same directory as the downloaded `docker-co
...
 docker-compose pull
 docker-compose up --detach
 ```
+
+
+## DigitalOcean
+
+Athens the team has tested the backend beta server on DigitalOcean the most, and not as much yet on other cloud providers like AWS or GCP.
+
+A minimum of 4gb of memory are needed.
```
Comment (filipesilva) [inline README.self-hosted.md:125]: Nit: `are needed` -> `is needed`

File: README.self-hosted.md [hunk 3]
```diff
@@ -100,10 +105,34 @@ or via an `.env` file located in the same directory as the downloaded `docker-co
...
+
+![image](https://user-images.githubusercontent.com/8952138/141150237-dce5f183-f25c-4a9b-9526-dcc310c09a44.png)
+
+Use marketplace docker image: `docker 19.03.12 on Ubuntu 20.04` which has docker and docker-compose pre-installed.
+
+![digital-ocean-docker-image](https://user-images.githubusercontent.com/8952138/141150155-7be8bce5-1804-431c-9cd0-4cf11e9c8d87.jpg)
+
+If you resize your droplet image, if you have trouble opening the console, just wait a few minutes and refresh the Digital Ocean dashboard.
```
Comment (filipesilva) [inline README.self-hosted.md:133]: Nit: `if you have trouble` -> `or if you have trouble`

File: README.self-hosted.md [hunk 4]
```diff
@@ -100,10 +105,34 @@ or via an `.env` file located in the same directory as the downloaded `docker-co
...
+
+Once your droplet has been created, open the console to ssh to the server, follow the instructions in [Docker](#docker) to start the server, and copy and paste the IP address to your Athens client to connect to the server (and add a password if you configured one).
+
+![Athens Vision _ Ecosystem Visualization _ Product - Frame 13](https://user-images.githubusercontent.com/8952138/141150925-9f8df004-faa0-4fbe-9875-c276d60c5118.jpg)
```
Comment (filipesilva) [inline README.self-hosted.md:137]: Is this image name right?

**Review by filipesilva — APPROVED** [review]
LGTM, just a bit worried about giving such broad `chmod` instructions. If we can narrow it down a bit it would be better.

**Review by filipesilva — APPROVED** [review]

**Pull request merged**